agent/pkg/models/models.go

```diff
@@ -16,15 +16,9 @@ func GetEntry(r *tapApi.MizuEntry, v tapApi.DataUnmarshaler) error {
 	return v.UnmarshalData(r)
 }
 
-type EntriesFilter struct {
-	Limit     int    `form:"limit" validate:"required,min=1,max=200"`
-	Operator  string `form:"operator" validate:"required,oneof='lt' 'gt'"`
-	Timestamp int64  `form:"timestamp" validate:"required,min=1"`
-}
-
 type WebSocketEntryMessage struct {
 	*shared.WebSocketMessageMetadata
-	Data *tapApi.BaseEntryDetails `json:"data,omitempty"`
+	Data map[string]interface{} `json:"data,omitempty"`
```

Comment:
why is this no longer a struct?

Reply:
It's changed with this commit https://github.com/up9inc/mizu/commit/4f74be47d3b57028aa3df69a2b5c6954b03be636

Since the `WebSocketEntryMessage` built as a result of database's response instead of the dissect and directly pass to UI approach in the past (which makes it unqueryable), we have a `map[string]interface{}` instead of a static data type:

https://github.com/up9inc/mizu/blob/b0db97d72fd1e7b396502ce312581e36f2f38c88/agent/pkg/api/socket_routes.go#L127-L131

It's possible to cast it to `*tapApi.BaseEntryDetails` through *unmarshal -> marshal -> unmarshal* but it's not required because database returns it pretty much unmodified.